pallets/scoring/src/types.rs

```diff
@@ -0,0 +1,95 @@
+// This file is part of CORD – https://cord.network
+
+// Copyright (C) 2019-2022 Dhiway Networks Pvt. Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// CORD is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// CORD is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with CORD. If not, see <https://www.gnu.org/licenses/>.
+
```

Comment:
All type names should align with the names mentioned in the document - https://docs.google.com/document/d/1PWAOC9gWo0fL-crNrM-A1uXwzwNRY9jCMyc5L20_xRk/edit#heading=h.fnzqmubsnl03